app/forms/CASForm.scala

```diff
@@ -33,6 +33,6 @@ object CASForm {
     "cas" -> mapping(
       "period" -> number(min = 1, max = 52).transform[Weeks](Weeks.weeks, _.getWeeks),
       "subscriptionCode" -> of[SubscriptionCode]
-    )(TokenPayload.apply)(t => Some(t.period, t.subscriptionCode))
+    )(TokenPayload(LocalDate.now))(t => Some(t.period, t.subscriptionCode))
```

Comment:
the library always needed the current date, but I took the opportunity to make it more of a pure function.  The only significant change is the library version really.